### ISSUE TITLE
[FLAKE] [sig-autoscaling] [Feature:HPA] Horizontal pod autoscaling (scale resource: CPU) [sig-autoscaling] [Serial] [Slow] ReplicationController Should scale from 1 pod to 3 pods and from 3 to 5 and verify decision stability

### ISSUE THREAD

**Issue by soggiest** [kubernetes/kubernetes#82287]
**Which jobs are failing**:
ci-kubernetes-e2e-aks-engine-azure-1-16-windows

**Which test(s) are failing**:
[sig-autoscaling] [Feature:HPA] Horizontal pod autoscaling (scale resource: CPU) [sig-autoscaling] [Serial] [Slow] ReplicationController Should scale from 1 pod to 3 pods and from 3 to 5 and verify decision stability

**Since when has it been failing**:
8/31/2019

**Testgrid link**:
https://testgrid.k8s.io/sig-release-1.16-informing#aks-engine-azure-1-16-windows

**Reason for failure**:

```
test/e2e/autoscaling/horizontal_pod_autoscaling.go:61
Sep  3 08:04:18.123: Unexpected error:
    <*errors.errorString | 0xc002102990>: {
        s: "number of replicas above target",
    }
    number of replicas above target
occurred
test/e2e/autoscaling/horizontal_pod_autoscaling.go:126
```

**Anything else we need to know**:

/milestone v1.16
/priority critical-urgent
/kind flaking-test
/cc @kubernetes/sig-autoscaling-bugs 
/cc @alejandrox1 @jimangel @alenkacz @Verolop

**Comment by marosset** [comment]
We are aware of the issue and investigating

/cc @bclau @adelina-t

**Comment by marosset** [comment]
@bclau was doing some investigation here and found the following:
The tests start containers which are configured to consume a specified CPU percentage for a specified amount of time and the these percentages are used to determine if horizontal pod scaling is behaving correctly. 
After some investigation we found that svchost.exe was also consuming a non-trivial amount of CPU (which is not accounted for in the tests) impacting horizontal scaling behavior.

I don't believe this should be a release blocker.

**Comment by lachie83** [comment]
Thanks for the udpate @marosset

**Comment by bclau** [comment]
Indeed, this shouldn't block the release, this can be fixed by updating the images used by the test. We're currently discussing the issue, but we can update the images in the meanwhile. Will update later with more information.

**Comment by bclau** [comment]
I've updated the "e2eteam/resource-consumer:1.5", which is being used in testing Windows. I've tried running it a couple of times in my env, and the test is now passing properly.

The base image used was ``mcr.microsoft.com/windows/servercore:ltsc2019``, which has some random CPU spikes caused by the container's ``svchost.exe``, which then results in more CPU being consumed than intended, leading to more replicas to be spawned than needed, causing the test to fail. Now, the base image has been changed to ``mcr.microsoft.com/windows/nanoserver:1809``, which doesn't seem to have this issue.

Nonetheless, the ``servercore`` image will have to be investigated to determine the reason why ``svchost.exe`` spikes.

Additionally, because of this reason, centralizing the ``resource-consumer`` image into ``agnhost`` will have to shelved for now, until the issue is resolved.

**Comment by PatrickLang** [comment]
@soggiest - this seems to be working well now, are you ok to close?

**Comment by soggiest** [comment]
We've had successes in the test since 09/04 but there was a failure yesterday. I'm putting this under observation until we see regular successes
